spec/lib/carma/client/mule_soft_auth_token_client_spec.rb

```diff
@@ -0,0 +1,85 @@
+# frozen_string_literal: true
+
+require 'rails_helper'
+require 'carma/client/mule_soft_auth_token_client'
+
+class MockFaradayResponse
```

Comment:
I could DRY This up and put it in a shared helper or something? I'm not sure if there's a better way here so any advice is appreciated.